python/dgl/graphbolt/sampled_sub_graph.py

```diff
@@ -0,0 +1,172 @@
+"""Heterogeneous sampled subgraph."""
+# pylint: disable= invalid-name
+from collections.abc import Mapping
+from typing import Dict, Tuple, Union
+
+import torch
+
+
+class SampledSubGraph:
+    r"""Class for sampled subgraph."""
+
+    def __init__(
+        self,
+        node_pairs: Union[
+            Dict[Tuple[str, str, str], Tuple[torch.tensor, torch.tensor]],
+            Tuple[torch.tensor, torch.tensor],
+        ],
+        reverse_column_node_ids: Union[
+            Dict[str, torch.tensor], torch.tensor
+        ] = None,
+        reverse_row_node_ids: Union[
```

Comment:
Since we are not in the scope of a `CSC` graph, it is strange to use `row` and `column`. Can we use `src` and `dst`?

Reply:
Actually we don't want to limit the format of the sub graph, it could be a `CSC`, `CSR` or `COO`, and all of them can be represented as 2 tensors. From this perspective,  it's better to use `row` and `column`?